src/services/itemMembership/plugins/MembershipRequest/repository.ts

```diff
@@ -7,4 +7,30 @@ export class MembershipRequestRepository extends AbstractRepository<MembershipRe
   constructor(manager?: EntityManager) {
     super(MembershipRequest, manager);
   }
+
+  async get(memberId: string, itemId: string) {
+    return await this.repository.findOne({
+      where: {
+        member: { id: memberId },
+        item: { id: itemId },
+      },
+      relations: ['member', 'item'],
+    });
+  }
+
+  async post(memberId: string, itemId: string) {
+    await this.repository.insert({
+      member: { id: memberId },
+      item: { id: itemId },
+    });
+    const result = await this.get(memberId, itemId);
+    return result;
+  }
+
+  async getAllByItem(itemId: string) {
```

Comment:
Maybe `getByItem` is enough here ? But if you prefer `getAllByItem` it's ok for me ^^